### ISSUE TITLE
Parse error in CUE sheets lacking double quotes for string values

### ISSUE THREAD

**Issue by litemotiv** [Taiko2k/Tauon#1207]
Version: flatpak 7.8.0

CUE sheets that lack double quotes for string values will cause a parse error, leading them to be omitted. 

Example, the title of track 02 lacks double quotes and will be omitted during parsing:

```
PERFORMER "Various Artists"
TITLE "Le Café Abstrait, Volume 4: Soundtrack for a Moment"
REM MUSICBRAINZ_ALBUM_ID ef215a00-f4d0-4f38-9c8a-505e52a393e5
REM MUSICBRAINZ_ALBUM_ARTIST_ID 89ad4ac3-39f7-470e-963a-56509c546377
REM DATE 2003-07
FILE "Le Cafe abstrait Vol. 4.flac" WAVE
  TRACK 01 AUDIO
    PERFORMER "Lee Johnson"
    TITLE "Ancient Sound Pictures"
    REM MUSICBRAINZ_TRACK_ID f55b8b75-17ad-3f52-8f71-b57bfe8b2671
    REM MUSICBRAINZ_ARTIST_ID 43baca47-604b-4b33-a405-5ea779c51a4c
    INDEX 01 00:00:00
  TRACK 02 AUDIO
    PERFORMER "Julian Petrian"
    TITLE Embrace
    REM MUSICBRAINZ_TRACK_ID 92c9ffd5-9265-3dad-8ca9-aedb948b521d
    REM MUSICBRAINZ_ARTIST_ID 16cbce18-52dc-4275-be5b-0f97aa1f5ebf
    INDEX 01 06:04:67
```

**Comment by Taiko2k** [comment]
OK Ive committed a fix for this, this will be available in the next release.

**Comment by C0rn3j** [comment]
Looks like this already landed with [7.8.1](https://github.com/Taiko2k/Tauon/releases/tag/v7.8.1) and was just forgotten to be closed.
